File: buzzvil-sdk-v5-ios/buzzvil-sdk-ios-objc/Custom/CustomBrowserViewController.m
#import "CustomBrowserViewController.h"
@import BuzzvilSDK;

@interface CustomBrowserViewController ()

@property (nonatomic, strong, readonly) UIViewController *browserViewController;

@end

@implementation CustomBrowserViewController

- (void)viewDidLoad {
  [super viewDidLoad];
  
  _browserViewController = [[BuzzAdBrowser sharedInstance] browserViewController];
  [self displayContentViewController:_browserViewController];
}

- (void)displayContentViewController:(UIViewController *)contentViewController {
  [self addChildViewController:contentViewController];
  contentViewController.view.frame = self.view.bounds;
  [self.view addSubview:contentViewController.view];
  [contentViewController didMoveToParentViewController:self];
}

@end

File: archive/buzzvil-sdk-v5-ios/buzzvil-sdk-ios-swift/Custom/CustomBrowserViewController.swift
import UIKit
import BuzzvilSDK

class CustomBrowserViewController: UIViewController {
  private lazy var browserViewController: UIViewController = {
      BuzzAdBrowser.sharedInstance().browserViewController()
    }()
    
    override func viewDidLoad() {
      super.viewDidLoad()
      
      displayContentViewController(browserViewController)
    }

    private func displayContentViewController(_ contentViewController: UIViewController) {
      addChild(contentViewController)
      contentViewController.view.frame = view.bounds
      view.addSubview(contentViewController.view)
      contentViewController.didMove(toParent: self)
    }
}
